### ISSUE TITLE
any tips about how to train my own dataset

### ISSUE THREAD

**Issue by ronghui19** [NVlabs/stylegan2-ada-pytorch#25]
i have 2000+ images.  image resolution is 256x256. 
My configuration is as such: 
 --kimg=25000 --augpipe=bgcfn --gpus=2 --resume=ffhq256 --cfg=paper256 --snap=10

fid value is around 60. it's difficult to decrease fid value. any suggestion to decrease it?

**Comment by futscdav** [comment]
It would be great if we could get the loss plots from reference model training to get a better idea of what looks right.

**Comment by woctezuma** [comment]
You need to tell us more about your dataset.

Other than that, the `augpipe` could be more constrained.

**Comment by Lover19** [comment]
No thanks , Must Police am 15.2.2021
No Interesse 

Nokia


Am 10.02.2021 um 09:57 schrieb Wok <notifications@github.com>:

﻿
You need to tell us more about your dataset.

Other than that, the augpipe could be more constrained.

**Comment by NNDam** [comment]
I've trained stylegan2-ada model from scratch with default config for 1M face images 128x128, fid50k_full after 25000 kimgs is about 2.887, 125k face images 256x256 fid50k_full is about 5.424 (all face have the same alignment method), but fineturn with ffhq (different face alignment method) make fid50k_full > 10.
